scripts/deploy.js

```diff
@@ -105,6 +105,12 @@ function updateMetadata() {
         params = {
           ContentEncoding: "gzip",
         };
+        if (filepath.includes("service-worker.js")) {
+          params = {
+            ContentEncoding: "gzip",
+            CacheControl: "max-age=0",
```

Comment:
Do we want to use the full set of cache-busting parameters here (i.e. `no-cache, must-revalidate, max-age=0`)? I feel like I encountered scenarios in the past where having all those parameters was needed to prevent caching at all the various levels (locally, CDN).

But then there's a lot about caching which I don't fully understand yet...